apps/meteor/server/services/team/service.ts

```diff
@@ -1053,4 +1053,33 @@ export class TeamService extends ServiceClassInternal implements ITeamService {
 
 		return rooms;
 	}
+
+	// Returns the list of rooms and discussions a user has access to inside a team
+	// Rooms returned are a composition of the rooms the user is in + public rooms + discussions from the main room (if any)
+	async listChildren(
+		userId: string,
+		team: ITeam,
```

Comment:
```suggestion
		team: AtLeast<ITeam, '_id' | 'roomId'>,
```
No need to require the whole team document here if we're only using two attributes.